src/hooks/members/utils.ts

```diff
@@ -39,6 +39,10 @@ export const hasSomeRole = (
   return includesCustomPermissions || isInFilters;
 };
 
+export const truncateAddress = (name: string) => {
+  return `${name.slice(0, 6)}...${name.slice(-4)}`;
+};
```

Comment:
We already have a util for this, `splitAddress` which is more robust.

Please use that one.